Description: Removing codes which fetch data from external websites at runtime.
--- a/README.rdoc
+++ b/README.rdoc
@@ -3,7 +3,7 @@
 home :: https://github.com/ged/ruby-pg
 docs :: http://deveiate.org/code/pg
 
-{<img src="https://badges.gitter.im/Join%20Chat.svg" alt="Join the chat at https://gitter.im/ged/ruby-pg">}[https://gitter.im/ged/ruby-pg?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge]
+{Join the chat at https://gitter.im/ged/ruby-pg}[https://gitter.im/ged/ruby-pg?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge]
 
 
 == Description
@@ -30,8 +30,8 @@
 
 == Build Status
 
-{<img src="https://travis-ci.org/ged/ruby-pg.svg?branch=master" alt="Build Status Travis-CI" />}[https://travis-ci.org/ged/ruby-pg]
-{<img src="https://ci.appveyor.com/api/projects/status/gjx5axouf3b1wicp?svg=true" alt="Build Status Appveyor" />}[https://ci.appveyor.com/project/ged/ruby-pg-9j8l3]
+{Build Status Travis-CI}[https://travis-ci.org/ged/ruby-pg]
+{Build Status Appveyor}[https://ci.appveyor.com/project/ged/ruby-pg-9j8l3]
 
 
 == Requirements
